harmoni_dialogues/harmoni_bot/scripts/start_rasa_server.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+RASA_ASSISTANT=$(rosparam get /rasa/default_param/rasa_assistant)
+RASA_DIR=$(rosparam get /harmoni_bot_dir)
+
+if [ "$RASA_ASSISTANT" == "rasa_example" ] || [ "$RASA_ASSISTANT" == "rasa_greeter" ]
+  then
+    cd "$RASA_DIR"/src/"$RASA_ASSISTANT"
+    rasa train && rasa run
+else
```

Comment:
This check works for now, but it doesn't work with future bots. Can we dynamically support based on what directories exist in the RASA_DIR?